tests/test_modules.py

```diff
@@ -558,10 +558,14 @@ def forward(self, x):
         return self.activation(self.linear(self.initializer + x)) * self.scale
 
 
-def make_test(fn, expected_ops=None):
+def make_test(fn, expected_ops=None, expected_frame_count=4):
```

Comment:
Why is the 4 needed?  Are we sure it is right?  I don't think most of these should be swapping out parameters.